README.md

```diff
@@ -1,8 +1,251 @@
+    codeset (list|register|get|delete)
+    runnable (list|register|get)
+    workflow (list|register|get|assign|list-runs)
+
+Additional help:
+    bin/fuseml_core-cli SERVICE [ENDPOINT] --help
+
+Example:
+    bin/fuseml_core-cli application list --type "predictor" --workflow "mlflow-sklearn-e2e"
+    bin/fuseml_core-cli codeset list --project "mlflow-project-01" --label "mlflow"
+    bin/fuseml_core-cli runnable list --id "ml-trainer-123" --kind "trainer" --labels '{
+          "function": "predict|train",
+          "library": "pytorch"
+       }'
+    bin/fuseml_core-cli workflow list --name "workflow"
+```
+
+  Without `-url` argument, client tries to connect to the server running locally at your machine. To find the URL of the fuseml-core server service that is installed in your Kubernetes cluster, call 
+
+  ```bash
+  kubectl get VirtualService -n fuseml-core fuseml-core -o jsonpath="{.spec.hosts[0]}"
+  ```
+  and pass the value to each client call with `-url` option.
+
+
+  From the usage output you can see that with the client you can operate with various FuseML artefacts.
+
+  * Codesets describe the code of your ML application, for example MLflow project. They are currently implemented as git repositories.
+
+    Create new codeset by calling `bin/fuseml_core-cli codeset register` command. use the `--help` option to get the description of the command usage.
+
+    Example:
+    ```bash
+    bin/fuseml_core-cli codeset register -name "test" --project "mlflow-project-01" --location "/tmp/mlflow/mlflow-01"
+    ```
+
+    `--location` argument points to the directory on your machine where your ML application code is located.
+
+    After registering, use
+    ```
+    bin/fuseml_core-cli codeset list
+    ```
+    command to list available codesets and check the value of "URL" in the output of your newly registered codeset. Use this value to git clone the code into another directory. Now you can work in this directory just like with any other project saved in git. Assuming you have configured a workflow assignment (see parts about workflows), any time you commit and push new changes to your code, a worklflow run will be executed and you can expect updated application as a result.
+
+    Note: the `codeset list` command allows filtering the output by project or user defined labels.
+
+
+  * Workflows define the full AI/ML workflow. In short, this could be described as a way to process the input (the Codeset) and turn it into the output application (e.g. ML predictor).
+
+    For registering new workflow, use
+    ```bash
+    bin/fuseml_core-cli workflow register
+    ``` command.
+
+    After the workflow is registered, you should assign a codeset to it. That way the workflow will be automatically executed every time you push a new change to your git repository that is represented by a codeset.
+
+    ```bash
+    bin/fuseml_core-cli workflow assign -name "workflow-name" -codeset-name "test" -codeset-project "mlflow-project-01"
```

Comment:
might be worthy mentioning that when assigning a workflow to a codeset, the workflow is also triggered.

Reply:
OK, I'll add it to the line above.